src/Squirrel/IUpdateManager.cs

```diff
@@ -167,8 +167,11 @@ public static async Task<ReleaseEntry> UpdateApp(this IUpdateManager This, Actio
                 await This.ErrorIfThrows(() => 
                     This.CreateUninstallerRegistryEntry(),
                     "Failed to set up uninstaller");
-            } catch {
-                if (ignoreDeltaUpdates == false) {
+            } catch (IOException ex) when ((ex.HResult & 0xFFFF) == 0x27 || (ex.HResult & 0xFFFF) == 0x70) {
```

Comment:
Can you maybe comment on those magic numbers or add a doc link explaining the codes?

Reply:
Sure, I've added both.